### DISCUSSION TITLE
Enter Fullscreen Mode When Loading a Rom

### DISCUSSION THREAD

**Discussion by DeathWrench** [project64/project64#2265] [Ideas]
If you click outside of the window while a rom is loading, fullscreen no longer hides the mouse, and is now visible fullscreen mode, it now hides behind window when not in fullscreen. If you alt tab and tab back into PJ64 the bar at the top appears. If you try to alt tab while it is in windowed mode in this state, it will return to fullscreen with the bar on top.

here is a video: https://youtu.be/P0911UkjofI

**Comment by dftf-stu** [comment]
@DeathWrench: I can see from the video you're using version 2.4.0 of _Project64_.

If you use the latest stable release (3.0.1) from [here](https://www.pj64-emu.com/public-releases), or the latest alpha/beta release from [here](https://www.pj64-emu.com/nightly-builds), does the issue still occur?  If so, are you using a video plug-in that is not one of the ones the app comes with built-in?

**Reply by DeathWrench** [reply]
I'm on a 4.0+ build right now and it seems to be fixed. 
![image](https://cdn.discordapp.com/attachments/515842332201975821/1023006422746791957/unknown.png)
